snapcraft/tests/commands/test_help.py

```diff
@@ -69,6 +69,15 @@ def test_print_module_help_for_valid_plugin(self):
                          'The help message does not start with {!r} but with '
                          '{!r} instead'.format(expected, output))
 
+    def test_print_module_with_dashes_help_for_valid_plugin(self):
+        main(['help', 'plainbox-provider'])
+
+        expected = ' Create parts containing'
+        output = self.fake_terminal.getvalue()[:len(expected)]
+        self.assertEqual(output, expected,
+                         'The help message does not start with {!r} but with '
+                         '{!r} instead'.format(expected, output))
```

Comment:
There's a testtools matcher that you can use here to make it simpler:

https://testtools.readthedocs.io/en/latest/for-test-authors.html#startswith-endswith